index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Portfolio | Edwin Gichuhi</title>
+    </head>
+
+    <body>
+        <nav class="toolbar">
+            <a href="#">John Doe</a>
+            <img src="/images/Icon-Menu.png" alt="menu">
+        </nav>
+        <section class="headline">
+            <h1>Hey There. I'm Alex</h1>
+            <h2>I’am a Software Developer</h2>
+            <h3>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you
+                see and have a project you need coded,
+                don’t hestiate to contact me.</h3>
+            <section class="socials">
+                <img src="/images/Icon-GitHub.png" alt="Github">
+                <img src="/images/Icon-Linkedin.png" alt="Github">
+                <img src="/images/Vector.png" alt="Github">
+                <img src="/images/Icon-Twitter.png" alt="Github">
+                <img src="/images/Icon-Meta.png" alt="Github">
```

Comment:
:red_circle: This list of icons should use a `ul` tag to group them,  each icon should be in `li` tag; each icon should be in   anchor tag  link as follows :arrow_down: 
```
<ul  class="socials">
   <li><a> <img src="/images/Icon-Meta.png" alt="Github"></a></li>
   ....
   ....
</ul>
```
**NOTE:  The a tag will let you connect your social networks with the icons, using href.**